lib/view_model/send/send_view_model.dart

```diff
@@ -530,6 +531,17 @@ abstract class SendViewModelBase extends WalletChangeListenerViewModel with Stor
       throw Exception('Priority is null for wallet type: ${wallet.type}');
     }
 
+    if (hasCoinControl) {
+      final vm = getIt.get<UnspentCoinsListViewModel>(param1: coinTypeToSpendFrom);
```

Comment:
please add this as a dependency in di.dart for the send_view_model
so that if we modify this, it's gonna be only 1 file to look into